### PR TITLE
feat(tabs): added secondary border-bottom variation, update demos

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/4753

Link to the updated demo - https://patternfly-pr-4774.surge.sh/components/tabs/html-demos#open-tabs-with-secondary-tabs

@mceledonia @mcarrano are there others we should update? I can give you a list of everywhere secondary tabs is used if that helps.

### CODE REVIEW

**Review by mcarrano — COMMENTED** [review]
I guess this looks OK, but I will defer to @mceledonia .

**Review by mceledonia — APPROVED** [review]
Looking good, thanks!

**Review by mattnolting — APPROVED** [review]
LPTM 🚀

File: src/patternfly/components/TabContent/examples/TabContent.md [hunk 1]
```diff
@@ -23,16 +23,16 @@ cssPrefix: pf-c-tab-content
 
 ### Padding
 ```hbs
-{{#> tab-content tab-content--modifier="pf-m-padding" tab-content--IsActive="true" tab-content--attribute='id="tab1-panel"'}}
+{{#> tab-content tab-content-body--modifier="pf-m-padding" tab-content--IsActive="true" tab-content--attribute='id="tab1-panel"'}}
```
Comment (mattnolting) [inline src/patternfly/components/TabContent/examples/TabContent.md:26]: 🔥

File: src/patternfly/demos/Tabs/tabs-template-pod-tab-list.hbs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {{#if tabs-template-pod-tab-list--IsSecondary}}
-  {{#> tabs tabs-id=tab-template--id tabs--modifier="pf-m-inset pf-m-secondary" tabs--attribute=tabs-template-pod-tab-list--attribute}}
+  {{#> tabs tabs-id=tab-template--id tabs--modifier=(concat "pf-m-secondary " tabs-template-pod-tab-list--modifier) tabs--attribute=tabs-template-pod-tab-list--attribute}}
```
Comment (mattnolting) [inline src/patternfly/demos/Tabs/tabs-template-pod-tab-list.hbs:2]: 🔥

**Review by mattnolting — APPROVED** [review]
Nice work! LPTM 🤩

**Pull request merged**